### ISSUE TITLE
sparkR kernel not responding

### ISSUE THREAD

**Issue by rahulsingh303** [jupyter-incubator/sparkmagic#328]
I was able to use other kernels other than sparkR kernel , also I dont see any examples for that in examples section . Is it suppose to work ? When I run any command using sparkR kernel the kernel just shows busy in jupyter and never shows any output ? Any idea what else need to be done ?

**Comment by aggFTW** [comment]
Yes, it's supposed to work. Did you configure the spark_r kernel too?

```
  "kernel_r_credentials" : {
    "username": "",
    "password": "",
    "url": "http://localhost:8998"
  }
```

I think that config + samples are not updated on the repo. @jeffersonezra could you please add them?

**Comment by rahulsingh303** [comment]
working fine now , thanks for quick response.

**Comment by aggFTW** [comment]
No worries. I'm glad you got it to work 👍
